docs/source/apps/projinfo.rst

```diff
@@ -285,6 +286,16 @@ The following control parameters can appear in any order:
     If also specifying a CRS object and the :option:`--output-id` option, the
     definition of the object as SQL statements will be appended.
 
+.. option:: --list-crs [list-crs-filter]
+
+    .. versionadded:: 8.1
+
+    Outputs a list (authority name:code and CRS name) of the filtered CRSs from the database.
+    If no filter is provided all authority names and types of non deprecated CRSs are dumped.
+    list-crs-filter is a comma separated combination of: allow_deprecated,geodetic,geocentric,
+    geographic,geographic_2d,geographic_3d,vertical,projected,compound.
+    Affected by options `--authority`, `--bbox`, `--main-db-path` and `--aux-db-path`    
```

Comment:
```suggestion
    Affected by options :option:`--authority` and :option:`--bbox`
```
It is kind of implied that --main-db-path and --aux-db-path affect the behavior of the whole utility.